x-pack/test/idp-fixture/build.gradle

```diff
@@ -1,4 +1,36 @@
+import org.elasticsearch.gradle.VersionProperties
+import org.elasticsearch.gradle.Architecture
+
 apply plugin: 'elasticsearch.build'
```

Comment:
I wonder if we can remove `elasticsearch.build` plugin. This project contains no code or tests, it's simply a test fixture.